contracts/upgradeable_contracts/U_ForeignBridge.sol

```diff
@@ -40,6 +42,8 @@ contract ForeignBridge is ERC677Receiver, Validatable {
         uintStorage[keccak256("deployedAtBlock")] = block.number;
         uintStorage[keccak256("maxPerTx")] = _maxPerTx;
         uintStorage[keccak256("minPerTx")] = _minPerTx;
+        uintStorage[keccak256("gasPrice")] = _homeGasPrice;
```

Comment:
I think there is a mistake here.
Initially foreign contract operated with a gas price to send it as part of `Withdraw()` event to notify home side how much a user would like to compensate validators for relaying his/her transaction. It was assumed that gas price was specified by the user as part of the withdraw request. We do not use the compensation but didn't have resources to change Rust code to not operate with gas prices received in `Withdraw()` event, so we hardcoded this value to 1 gwei. Moreover this value does not need to be returned through getter since there is no scenario when someone would like to know which gas price is used for this _"legacy"_ code.

Now we are introducing new entity - a gas price to be requested by the bridge instance to use it in its transaction. This is completely different use case. 

Here is my suggestion:
1. to have two entities: `gasPriceForCompensationAtHomeSide` and `gasPrice`
2. `gasPriceForCompensationAtHomeSide` will be always 1gwei. We will not allow to change it. It is used as a stub in `Withdraw()` event.
3. `gasPrice` will be configured at initialization stage and could be changed by `setGasPrice()`. It is used by the bridge to set gas price for transactions if the oracle is not accessible.

Reply:
ok